crates/runtime/src/lib.rs

```diff
@@ -83,6 +83,8 @@ mod module_id;
 pub use module_id::{CompiledModuleId, CompiledModuleIdAllocator};
 
 mod cow;
+pub mod parking_spot;
```

Comment:
I think it should be ok to drop the `pub` here

Reply:
removed